packages/cli/src/config/config.ts

```diff
@@ -245,6 +245,7 @@ export async function loadCliConfig(
     bugCommand: settings.bugCommand,
     model: argv.model!,
     extensionContextFilePaths,
+    hideTips: settings.hideTips ?? false,
```

Comment:
is this really needed in config or is just settings sufficient?

Reply:
Technically, settings is sufficient, and that’s how I initially did it. However, I later noticed that there is this „config” abstraction which kind of pulls all configuration sources together and it seemed like I should be using that. If not, I can go back to the (simpler) settings based approach, just let me know

Reply:
Lets just go with settings. You can always add it more places later.  Generally I would only put it in settings if there isnt a compelling reason why it should be settable on the command line.